integration-cli/docker_cli_rmi_test.go

```diff
@@ -229,51 +183,32 @@ func (s *DockerSuite) TestRmiWithMultipleRepositories(c *check.C) {
 	newRepo := "127.0.0.1:5000/busybox"
 	oldRepo := "busybox"
 	newTag := "busybox:test"
-	out, _, err := dockerCmdWithError("tag", oldRepo, newRepo)
-	if err != nil {
-		c.Fatalf("Could not tag busybox: %v: %s", err, out)
-	}
+	dockerCmd(c, "tag", oldRepo, newRepo)
 
-	out, _, err = dockerCmdWithError("run", "--name", "test", oldRepo, "touch", "/home/abcd")
-	if err != nil {
-		c.Fatalf("failed to run container: %v, output: %s", err, out)
-	}
+	dockerCmd(c, "run", "--name", "test", oldRepo, "touch", "/home/abcd")
 
-	out, _, err = dockerCmdWithError("commit", "test", newTag)
-	if err != nil {
-		c.Fatalf("failed to commit container: %v, output: %s", err, out)
-	}
+	dockerCmd(c, "commit", "test", newTag)
 
-	out, _, err = dockerCmdWithError("rmi", newTag)
-	if err != nil {
-		c.Fatalf("failed to remove image: %v, output: %s", err, out)
-	}
-	if !strings.Contains(out, "Untagged: "+newTag) {
-		c.Fatalf("Could not remove image %s: %s, %v", newTag, out, err)
-	}
+	out, _ := dockerCmd(c, "rmi", newTag)
+	c.Assert(out, checker.Contains, "Untagged: "+newTag)
 }
 
 func (s *DockerSuite) TestRmiBlank(c *check.C) {
 	testRequires(c, DaemonIsLinux)
 	// try to delete a blank image name
 	out, _, err := dockerCmdWithError("rmi", "")
-	if err == nil {
-		c.Fatal("Should have failed to delete '' image")
-	}
-	if strings.Contains(out, "no such id") {
-		c.Fatalf("Wrong error message generated: %s", out)
-	}
-	if !strings.Contains(out, "image name cannot be blank") {
-		c.Fatalf("Expected error message not generated: %s", out)
-	}
+	// Should have failed to delete '' image
+	c.Assert(err, checker.NotNil)
```

Comment:
Could you move the comment in the end of the line or in a `check.Commentf(…)` so we don't loose the "context" when this assertion fails ?
(and same on the lines above)